### ISSUE TITLE
macOS dock icon bounces until private messages or mentions have been read

### ISSUE THREAD

**Issue by ckojeder** [zulip/zulip-desktop#925]
On macOS the dock icon bounces indefinitely when you get a private message / mention unless you explicitly open the application window and navigate to private messages / mentions which caused the notification.

In my opinion it should only bounce once for a message / mention (or at least stop bouncing once you switch to Zulip and not require users to actually navigate to the private messages / mentions) :).

**Comment by manavmehta** [comment]
@akashnimare As, the work has been initiated in #733 and [#12222](https://github.com/zulip/zulip/pull/12222), are we waiting for some other feature that this one depends on? As [#12222](https://github.com/zulip/zulip/pull/12222) already implements the same in electron bridge

**Comment by akashnimare** [comment]
@ckojeder did you try "Bounce dock on new private message" setting? -

![image](https://user-images.githubusercontent.com/2263909/81642930-2a7fea80-9442-11ea-8cdf-4febbf10300d.png)

**Comment by ckojeder** [comment]
Yes, I did! This disables the bouncing animation completely but that's not what I'd like to see :).

Usually I think macOS messaging applications bounce the dock icon once if a notification is triggered and then stop.

In the Zulip desktop app the icon bounces indefinitely until all notifications have been cleared in the application - even switching to the application is not enough. You really have to navigate to the messages which triggered the notifications to end the bouncing animation - this is pretty annoying behaviour I think :).
